sections/exchange/hooks/useExchange.tsx

```diff
@@ -559,6 +561,14 @@ const useExchange = ({
 		return null;
 	}, [feeAmountInBaseCurrency, basePriceRate]);
 
+	useEffect(() => {
+		setCurrencyPair({
+			base: null,
+			quote: null,
+		});
+		// eslint-disable-next-line	react-hooks/exhaustive-deps
+	}, [network.id]);
```

Comment:
it wants me to add setCurrencyPair to the deps but this causes this `useEffect` to run excessively and eventually crashes the page with too many queries to local storage 

(we store selected synths in local storage to persist state between refreshes)

Reply:
can we cache `setCurrencyPair` via `useCallback` and appease the linter? does that work?

Reply:
ended up fixing the issue upstream 

https://github.com/Synthetixio/kwenta/pull/469/files#diff-c2167a32a8bb0c5741361d7e5ea6df1926f47c932438f401c5fb46bd2b9049f3R7